packages/v4/patternfly-docs/content/design-guidelines/components/text-input-group/text-input-group.md

```diff
@@ -3,8 +3,6 @@ id: Text input group
 section: components
 ---
 
-A **text input group** is a more flexible composable version of a [text input](/components/text-input). It enables consumers of PatternFly to build custom inputs for filtering and similar use cases by placing elements like icons, [chip groups](/components/chip-group) and [buttons](/components/button) within a text input.
```

Comment:
For some reason the chip groups and button links are dead for me. Looks like the link is directing to : /components/text-input-group/components/button, instead of just /components/button

Reply:
Thanks - they were missing the leading slash before `components` in the link